lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
```

Comment:
a ```<nav>``` element needs an ```aria-label``` element to make it feel special.  Without one, it just gets listed as "navigation, navigation, navigation" with all the other nav elements.  Let's give this guy a name.  How about ```aria-label="Team"``` so it will be listed as "Team navigation" to screen reader users.